api_tests/lib_sdk/actors/actor.ts

```diff
@@ -745,21 +801,36 @@ export class Actor {
     }
 }
 
-function defaultLedgerDescriptionForAsset(asset: AssetKind): Ledger {
+function defaultLedgerKindForAsset(asset: AssetKind): LedgerKind {
     switch (asset) {
-        case AssetKind.Bitcoin: {
+        case AssetKind.Bitcoin:
+            return LedgerKind.Bitcoin;
+        case AssetKind.Ether:
+            return LedgerKind.Ethereum;
+        case AssetKind.Erc20:
+            return LedgerKind.Ethereum;
+    }
+}
+
+/**
+ * WIP as the cnd REST API routes for lightning are not yet defined.
+ * @param ledger
+ * @returns The ledger formatted as needed for the request body to cnd HTTP API on the lightning route.
```

Comment:
They are gonna be different though if you think back to what the spikes says. The whole point of making new routes is to avoid these nested objects.